icu4j/main/classes/core/src/com/ibm/icu/number/NumberFormatterImpl.java

```diff
@@ -198,11 +204,13 @@ private static MicroPropsGenerator macrosToMicroGenerator(MacroProps macros, Mic
         // compact notation overrides the middle modifier (micros.modMiddle)
         // normally used for the percent pattern.
         boolean isCldrUnit = !isCurrency
-            && !isBaseUnit
-            && (unitWidth == UnitWidth.FULL_NAME
+                && !isBaseUnit
+                && (unitWidth == UnitWidth.FULL_NAME
```

Comment:
Please don't change the indentation

Reply:
I think this is the correct indentation

Reply:
No it's not - there's an open parenthesis on line 208 and a close parenthesis on line 211, the old identation helped clarify this. This new indentation serves no useful purpose other than causing the reviewers to have to think more.

I'm undoing it.

Reply:
Undone with 08744e0867.